.pre-commit-config.yaml

```diff
@@ -0,0 +1,8 @@
+repos:
+  - repo: local
+    hooks:
+      - id: spotless-check
+        name: Spotless Check
+        entry: ./gradlew spotlessCheck
+        language: system
+        pass_filenames: false
```

Comment:
can we apply spotless first and then check? Just a suggestion. If the fixes can be automated might as well automate them in pre-commit hook.